test/5_run_target.vader

```diff
@@ -0,0 +1,56 @@
+Before:
+    function! RemoveFile(file)
+        if has('win32')
+            echo system("del /F /Q \"".a:file."\" >nul 2>&1")
+        else
+            echo system("rm -rf '".a:file."'")
+        endif
+    endfunction
+
+    function! RemoveCMakeDirs()
+        if has('win32')
+            for l:value in ['cmake-build-Release', 'cmake-build-Debug', 'cmake-build-RelWithDebInfo', 'cmake-build-MinSizeRel', 'cmake-build', 'build_test']
+                echo system("rd /S /Q \"".l:value."\" >nul 2>&1")
+            endfor
+        else
+            echo system("rm -rf cmake-build*")
+        endif
+    endfunction
+    cd test_proj
+    call RemoveFile("compile_commands.json")
+    call RemoveCMakeDirs()
+
+    let g:cmake_compile_commands_link=""
+    let g:cmake_compile_commands=0
+    let g:cmake_usr_args=""
+    let g:cmake_cxx_compiler=""
+    let g:cmake_c_compiler=""
+    let g:cmake_build_type=""
+    let g:cmake_install_prefix=""
+    let g:cmake_project_generator=""
+    let g:make_arguments=""
+    let g:cmake_build_target="all"
+    let g:cmake_build_dir=""
+    let g:cmake_change_build_command=1
+    let g:cmake_reload_after_save=0
+    let g:cmake_build_dir_prefix="cmake-build-"
+
+After:
+    call RemoveFile("compile_commands.json")
+    call RemoveCMakeDirs()
+    cd ..
+
+Execute (Should not find non-binary target):
+    silent! CMake
+    silent! CMakeBuild
+    let result = utils#cmake#getBinaryPath()
+    AssertEqual '', result
+
+Execute (Should find binary target):
+    CMake
+    CMakeBuild
+    CMakeSelectTarget test_app
+    let result = utils#cmake#getBinaryPath()
+    if !has('win32')
+    AssertEqual 'cmake-build-Release/app/test_app', result
+    endif
```

Comment:
Could you add tests on `cmake4vim#RunTarget()`? 
It will be good to check that it works correct for negative cases too.